app/components/lg-lion-search.js

```diff
@@ -54,9 +61,31 @@ export default Ember.Component.extend({
       params['name'] = selectedName;
     }
 
+    if (selectedDob && !Ember.isBlank(selectedDob)) {
```

Comment:
@redyaffle  As I was saying above I think it would be nice to push this logic into its own component called something like `lg-dob-search-selector`. this lg-lion-search component should just have 2 date variables, selectedDateStart and selectedDateEnd, and it can pass those to the component and then component will be in charge of setting them based on the user selection (null if the user selects all otherwise the 2 correct dates).